src/Microsoft.Diagnostics.Runtime/src/Utilities/Logging/DefaultLogger.cs

```diff
@@ -0,0 +1,19 @@
+namespace Microsoft.Diagnostics.Runtime.Utilities
```

Comment:
This isn't actually used for anything in ClrMD, it's just a disconnected class.

In general I'm not opposed to adding logging capabilities to ClrMD.  However, if I accept a change to add logging capabilities it would have to meet this criteria:

1.  It should go in its own pull request.
2.  The checkin to add a logger should also go through the runtime and add the actual calls to the logger itself.  I don't like disconnected interfaces/classes without actually using them.